fuse/opcode.go

```diff
@@ -99,6 +99,9 @@ func doInit(server *Server, req *request) {
 		CongestionThreshold: uint16(server.opts.MaxBackground * 3 / 4),
 		MaxBackground:       uint16(server.opts.MaxBackground),
 	}
+	if server.opts.MaxReadAhead != 0 {
+		out.MaxReadAhead = uint32(server.opts.MaxReadAhead)
```

Comment:
the libfuse code in do_init looks (in fuse_lowlevel.c) like this

```c
	if (arg->minor >= 6) {
		if (arg->max_readahead < se->conn.max_readahead)
			se->conn.max_readahead = arg->max_readahead;
```

I read that as saying it sets the  max_readahead to the max_readahead if the current value is lower than the desired max_readahead.

So I think that would translate into go as

```go
	if server.opts.MaxReadAhead != 0 && out.MaxReadAhead < uint32(server.opts.MaxReadAhead) {
		out.MaxReadAhead = uint32(server.opts.MaxReadAhead)
	}
```

Thoughts?